logger/log.go

```diff
@@ -38,13 +38,37 @@ type Logger struct {
 	fl         *fileLogger
 }
 
-// NewStdLogger creates a logger with output directed to Stderr
-func NewStdLogger(time, debug, trace, colors, pid bool) *Logger {
+type LogOption interface {
```

Comment:
Why is type public but not the method?

Reply:
That makes it so that the `LogOption` can be used in the `NewStdLogger` func, but limiting the interface to only be implemented by code within the `nats-server/logger` package.